packages/cli/src/lib/flags.ts

```diff
@@ -67,6 +67,12 @@ export const commonFlags = {
       char: 'e',
```

Comment:
I thought I had removed `char: 'e'`. I wonder if we can do that as a part of marking it as deprecated? (this common flag might not even be imported anywhere)